keystore/kms/aws/key_manager.go

```diff
@@ -43,9 +45,20 @@ func (k *KeyManager) CreateKey(ctx context.Context, metaData baseKMS.CreateKeyMe
 		return nil, err
 	}
 
-	return &baseKMS.KeyMetadata{
-		KeyID: *keyMetadata.Arn,
-	}, nil
+	// wait some time for alias to be active
+	for {
```

Comment:
hm... lets make some limit of tries? infinite loop is a bad choice for us imho, because keymaker doesn't handle signals for interruptions. Additionally lets log cases when we try one more time to know that this environment has such issues.